masterlist.yaml

```
@@ -3297,6 +3297,20 @@ plugins:
         itm: 35
         udr: 12
 
+  - name: 'Nec-Character_Clair.esp'
+    url: [ 'https://www.nexusmods.com/oblivion/mods/15569/' ]
+    msg:
+      - <<: *alreadyInX
+        subs: [ '**DACompilation_GO ESPs**' ]
+        condition: 'active("DACompilation_GO(_SI)?(_APN)?\.esp")'
+      - <<: *useInstead
+        type: say
+        subs: [ '[Sclerocephalus CM Partner Compilation](https://www.nexusmods.com/oblivion/mods/33739/)' ]
+        condition: 'not active("DACompilation_GO(_SI)?(_APN)?\.esp")'
```

This would be better, as the current messages don't make sense, due to the CM Partner requirement.
```suggestion
    msg:
      - <<: *alreadyInX
        subs: [ 'DAClair.esp' ]
        condition: 'active("DAClair.esp")'
```

---

ok, and move other messages to DAClair?

---

> 
> 
> ok, and move other messages to DAClair?

Yes, but in a different pull request.